Shopipy.CategoryManagement/Controllers/CategoryController.cs

```diff
@@ -7,19 +8,20 @@
 
 namespace Shopipy.CategoryManagement.Controllers;
 
+[Authorize]
 [ApiController]
 [Route("[controller]/{businessId}")]
 public class CategoryController(ICategoryService categoryService, IMapper mapper) : ControllerBase
 {
-    [HttpGet("categories")]
+    [HttpGet]
```

Comment:
uhh so we're changing it to only [controller] for the calls? did we decide that?

Reply:
It will be `Category/13jkljkjflkf`, `categories` was redundant.

Reply:
yes that makes sense. So should we change other controllers as well?

Reply:
let's not forget to write that down as a change of the contract

Reply:
I think @imbieras had something to say about this whole thing and how we will have our routes.

Reply:
We will need to add /business/ before everything for example...

Reply:
Leaving the routing problem until the project is almost complete